src/lib/Containers/Conversation.tsx

```diff
@@ -276,60 +288,85 @@ function sendConversationMessage(
   })
 }
 
-// TODO: Make this a pagination container instead of fetching 100 messages
-export default createRefetchContainer(
+// TODO Make this a pagination container instead of fetching 100 messages
+export default createPaginationContainer(
   Conversation,
-  graphql`
-    fragment Conversation_me on Me {
-      conversation(id: $conversationID) {
-        __id
-        id
-        from {
-          name
-          email
-          initials
-        }
-        to {
-          name
-          initials
-        }
-        messages(first: 200) @connection(key: "Conversation_messages") {
-          pageInfo {
-            hasNextPage
+  {
+    me: graphql.experimental`
+      fragment Conversation_me on Me {
+        conversation(id: $conversationID) @connection(key: "Conversation_messages") {
+          __id
+          id
+          from {
+            name
+            email
+            initials
           }
-          edges {
-            node {
-              impulse_id
-              is_from_user
-              body
-              ...Message_message
-            }
+          to {
+            name
+            initials
           }
-        }
-        items {
-          item {
-            ... on Artwork {
-              __typename
-              href
-              ...ArtworkPreview_artwork
+          messages(first: $count, after: $cursor) {
```

Comment:
This may need a `@argumentDefinitions(count: { type: "Int", defaultValue: 10 }, cursor: { type: "String", defaultValue: "" })` as well.